circle.yml

```diff
@@ -66,10 +66,6 @@ commands:
           name: "Test"
           command: |
             cmake --build ~/build --target test -- ARGS="-j4 --schedule-random --output-on-failure"
-            # Test statically linked end-to-end example
-            ~/build/examples/evmc-example-static
-            # Test dynamically loaded end-to-end example
-            ~/build/examples/evmc-example ~/build/examples/example_vm/libexample-vm.so
```

Comment:
All of these are still built, right? I think these artifacts are used by some integration.

Reply:
Yes. Now CTest handles execution of these lines.